api/src/pcapi/routes/backoffice/venues/forms.py

```diff
@@ -135,6 +135,7 @@ class Meta:
     )
     regions = fields.PCSelectMultipleField("Régions", choices=get_regions_choices())
     department = fields.PCSelectMultipleField("Départements", choices=area_choices)
+    only_validated_offerers = fields.PCSwitchBooleanField("Uniquement les structures validées")
```

Comment:
Suite au commentaire précédent, je pense que c'est cela que tu visais : 
```suggestion
    only_validated_offerers = fields.PCSwitchBooleanField("Uniquement les structures validées", default="checked")
```
(j'ai regardé les `PCSwitchBooleanField` pour trouver la bonne valeur alors qu'intuitivement j'aurais écrit `default=True` ; comportement à vérifier)

Reply:
J'avais exploré cette voie, mais ça ne marche pas. Le code derrière le template indique :` {{ " checked" if field.data }}` 
Il faut donc une valeur par défaut dans le champ `only_validated_offerers`.